docroot/sites/all/themes/ilr_theme/js/ilr_theme.js

```diff
@@ -105,11 +105,17 @@
         if (isSubsite()) {
           var menuText = $('.menu-block-ilr-primary-menu li').text();
           var title = $('.subsite-header').text();
+          var ilrAt70 = $('body').hasClass('ilr-at-70');
           $subsiteTitlePercentageWidth = Math.round(title.length / menuText.length * 100);
           var maxWidth = menuText.length * 15;
           $('.menu-block-ilr-primary-menu ul').css('max-width', maxWidth);
           if ($subsiteTitlePercentageWidth < 25) {
-            $subsiteTitlePercentageWidth = 35;
+            if (ilrAt70) {
```

Comment:
The js change allows the 70th logo to move closer to the center of the page.  How would we alter the js to achieve the same goal?